online-shopping-website/src/Components/ProductDetails.js

```diff
@@ -0,0 +1,128 @@
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import Stack from '@mui/material/Stack';
+import AddIcon from '@mui/icons-material/Add';
+import RemoveIcon from '@mui/icons-material/Remove';
+import {useParams} from 'react-router-dom';
+import Products from '../TestValues.json';
+
+class ProductButtons extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            quantity: 1,
+            show: true,
+            max: 5,
+            min: 0
+        };
+    }
+
+    IncrementItem = () => {
+        if (this.state.quantity != 10) {
+            this.setState({
+                quantity: this.state.quantity + 1
+            });
+        }
+    }
+    DecreaseItem = () => {
+        if (this.state.quantity != 1) {
+            this.setState({quantity: this.state.quantity - 1});
+        }
+    }
+
+    UpdateValue = (e) => {
+        this.setState({quantity: e.target.value});
+    }
+
+    get quantity() {
+        return this.state.quantity;
+    }
+
+    render() {
+        return (
+            <div className="ProductDetails-QuantityButtonsContainer">
+                <h3>Quantity</h3>
+                <Stack className="ProductDetails-QuantityButtonsStack" direction="row" spacing={1}>
+                    <Button className="ProductDetails-QuantityButtons" variant="contained"
+                            disabled={this.state.quantity == 10}
+                            onClick={this.IncrementItem}>
+                        <AddIcon/>
+                    </Button>
+                    <input className="inputne" value={this.state.quantity} onChange={this.UpdateValue}/>
+                    <Button className="ProductDetails-QuantityButtons" variant="contained"
+                            disabled={this.state.quantity == 1}
+                            onClick={this.DecreaseItem}>
+                        <RemoveIcon/>
+                    </Button>
+                </Stack>
+                <h5 className="ProductDetails-ProductLimitText">Limit of 10 items per product in cart.</h5>
+                <Button className="ProductDetails-CartButton" variant="contained" endIcon={<AddShoppingCartIcon/>}>
+                    Add to cart
+                </Button>
```

Comment:
Love the add to cart icon!! Just make sure the colours are all consistent with our theme, which is pinks and greens. Maybe all buttons should be green? Check out App.css and I have my old CSS style for the add to cart button, that could be useful!

Reply:
Shouldn't we make a general CSS class for the theme instead of adding it into each component one at a time?

Reply:
Yeah exactly, I already have the class ready but you can adjust it as you need!!